entities.py

```diff
@@ -96,21 +134,42 @@ def _search(cls, name):
                 return item
 
     def add(cls, item, qty):
+        """
+        Add the specified qty of item to the backpack's contents.
+
+        Args:
+            item (Item): Item object to be added to the contents of the backpack.
+            qty (int): Amount of the item to be added to the contents of the backpack.
+        """
```

Comment:
Maybe add a max backpack size attribute?

Reply:
And items should have a size attribute too. Or weight. However you wanna call it.